tests/testthat/test-prepInputs.R

```diff
@@ -989,6 +989,7 @@ test_that("load rdata in prepInputs", {
 
 
 test_that("assessDataType doesn't work", {
+  require(raster)
```

Comment:
use `library()` in tests, and detach the package on exit (see other tests).

Reply:
Ideally use:
```
      testInitOut <- testInit(libraries = "raster")
      on.exit({
        testOnExit(testInitOut)
      }, add = TRUE)
```
See many examples. This function (`testInit`) loads the named libraries but also turns off "ask" and "verbose" and has options for other things. With `testOnExit`, it cleans up everything that changed (unloads libraries, resets all changed options).